packages/App/src/onBoardingPages/NextButton.tsx

```diff
@@ -0,0 +1,89 @@
+import React, {FC, useEffect} from 'react';
+import {View, StyleSheet, Image, TouchableOpacity} from 'react-native';
+import Svg, {Circle} from 'react-native-svg';
+import Animated, {
+  useSharedValue,
+  useAnimatedProps,
+  withTiming,
+} from 'react-native-reanimated';
+import img from '../../Pics/right-arrow.png';
+
+interface Props {
+  page: number;
+  scrollTo: Function;
```

Comment:
Use function type literal instead. `Function` does not restrict any parameters or return values. Also `scrollTo` is not a good name for that function, as it hints on some value to scroll (like scroll to 20 dp from the top). The function does not take any params, so it might be better to say `scrollToNext`
```suggestion
  scrollToNext: () => void;
```